astropy/io/fits/connect.py

```diff
@@ -316,6 +303,8 @@ def _encode_mixins(tbl):
     """Encode a Table ``tbl`` that may have mixin columns to a Table with only
     astropy Columns + appropriate meta-data to allow subsequent decoding.
     """
+    import yaml  # noqa
```

Comment:
I don't think this is necessary any more

Reply:
Done!

Reply:
Did you forget to push the removal of the import, or was it necessary after all?

Reply:
forgot to push the removal. Done now.